package.json

```diff
@@ -54,7 +54,7 @@
     "multistream": "^4.0.0",
     "package-json-versionify": "^1.0.4",
     "parse-numeric-range": "^1.2.0",
-    "parse-torrent": "^7.1.3",
+    "parse-torrent": "^8.1.0",
```

Comment:
Please update parse-torrent to ^9.0.0 and magnet-uri to ^6.0.0

Reply:
Updated! 👍